userspace/libsinsp/plugin.cpp

```diff
@@ -712,6 +713,18 @@ void sinsp_plugin::validate_init_config_json_schema(std::string& config, std::st
 	}
 }
 
+void sinsp_plugin::set_config(const std::string& config)
+{
+	std::string conf = config;
+	validate_config(conf);
+
+	if(m_state && m_handle->api.set_config)
+	{
+		m_handle->api.set_config(m_state, conf.c_str());
+		m_state = NULL;
```

Comment:
```suggestion
```
Why is the state set to null?

Reply:
Oops! A leftover from some prior experiments